extensions/notebook/src/jupyter/jupyterServerInstallation.ts

```diff
@@ -257,16 +287,6 @@ export class JupyterServerInstallation implements IJupyterServerInstallation {
 					.on('close', async () => {
 						//unpack python zip/tar file
 						outputChannel.appendLine(msgPythonUnpackPending);
-						let pythonSourcePath = path.join(installPath, constants.pythonBundleVersion);
-						if (await utils.exists(pythonSourcePath)) {
-							try {
-								// eslint-disable-next-line no-sync
-								fs.removeSync(pythonSourcePath);
```

Comment:
Were there any issues with overwriting an existing install in the exact same place? (like if you manually set the New Install path to the 0.0.1 folder)

Reply:
If the new install path is set to the 0.0.1 folder then Python 3.6 installation gets overwritten. I'll add a check to make sure the new path and old path are not the same before prompting to remove, otherwise we'll be removing the new Python 3.8 installation.

Reply:
Actually, looking at the folder again, the Python 3.8 files are just added and not overwriting the existing install. So I'll have to check if the folder is the same as before and remove the 3.6 installation first.

Reply:
@chlafreniere  @corivera  In this scenario (user manually changes the folder to the 0.0.1) and in the future when we do another Python update (and the folder path stays the same), are we okay with just overwriting the old installation and not prompting the user for removing the old Python version? 
If we want to continue allowing the user to keep the old Python version then we would need to add the bundle version back into the path.